web-app/src/pages/SignIn/SignIn.tsx

```diff
@@ -95,9 +94,9 @@ const SignIn = () => {
               />
               <label>Se souvenir de moi</label>
             </div>
-            <a href="#">Mot de passe oublié ?</a>
+            <Link to={FORGOT_PASSWORD_PATH}>Mot de passe oublié ?</Link>
           </div>
-          <button disabled={loading}>{loading ? <Loader /> : "Valider"}</button>
```

Comment:
You delete the loading but you let the declaration l.27.

Reply:
ok